File: RickLocalizationAPI/Domain/Models/MortyModel.cs
namespace RickLocalizationAPI.Models
{
    public class MortyModel
    {
        public int Id { get; set; }

        public string Image { get; set; }

        public string Name { get; set; }

        public int Age { get; set; }

        public string Occupation { get; set; }

        public RickModel Rick { get; set; }

        public DimensionModel Dimension { get; set; }

    }
}

File: RickLocalizationApp/src/app/models/morty.model.ts
import { DimensionModel } from "./dimension.model";
import { RickModel } from "./rick.model";

export class MortyModel {
    id: number;
    image: string;
    name: string;
    age: number;
    occupation: string;
    rick: RickModel;
    dimension: DimensionModel;
}